tests/fixtures.py

```diff
@@ -57,6 +59,10 @@
 faker = Faker()
 integration_config = load_toml("fidesops-integration.toml")
 
+
+# Hide pytest 'cannot collect test class' warnings
+TestStatus.__test__ = False
```

Comment:
This (and TestCache, which I didn't fix due to a circular import issue) just throws pytest warnings since it's named "Test_" and pytest complains that it can't be built since it has an __init__ method. Can also be renamed.

Reply:
ah ok that makes sense, thanks for renaming!